src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -146,6 +147,14 @@ def get(self, request, organization):
                     response['X-Sentry-Direct-Hit'] = '1'
                     return response
 
+        # If group ids specified, just ignore any query components
+        group_ids = set(map(int, request.GET.getlist('group')))
```

Comment:
You should catch `ValueError` here in case the query parameter is empty or a non-int string.